docs/src/sections/sprints/sprint_9.adoc

```diff
@@ -1,5 +1,103 @@
+powiadomienia. W tym zadaniu napotkano problem z nawigacją pomiędzy ekranami. Aktualna implementacja nie pozwalała na
+utworzenie ekranu wyświetlanego nad innymi panelami, a tak właśnie zaplanowano implementację ekranu szczegółów
+powiadomienia. Wymagało to praktycznie przepisania kodu odpowiedzialnego za nawigację w aplikacji. W tym sprincie
+planowano również zadania wykańczające ekran szczegółów powiadomienia, czyli dodanie samych szczegółów oraz
+implementacja procesu potwierdzania i odrzucania powiadomienia. Zaplanowano również obsługę powiadomień _push_
+przychodzących z serwera. Niestety tych zadań nie udało się ukończyć i zostały one przeniesione do następnej iteracji.
+
+Dwa następne zadania dotyczą dalszych prac nad serwerem. Komponent ten był już bliski ukończenia, a zadań w rejestrze
+było coraz mniej. Pierwsze zadanie dotyczyło napisanie pliku _readme_ dla serwera, zawierającego informacje o
+komponencie oraz instrukcje jego uruchomienia i testowania. Drugie zadanie dotyczyło brakującej funkcjonalności
+profilu użytkownika. Dodane zostały dwa punkty końcowe. Ta funkcjonalność wymagała dodania możliwości pobrania
+szczegółów danego użytkownika. W pierwszym wydaniu zaplanowano, że użytkownik będzie mógł zmienić tylko i wyłącznie
+hasło, co zostało zaimplementowane w ramach drugiego punktu końcowego. Modyfikacja pozostałych pól jest w planach
+na następne iteracje.
+
+Następne zadanie dotyczyło dodania formularza tworzącego nowe transakcje. Na jego potrzeby dodano w ramach tego zadania
+dodatkowy punkt końcowy zwracający dane wymagane do tworzenia transakcji takich jak dostępne środki i wysokość opłaty
+transakcyjnej. Dodatkowo należało zaimplementować filtr uniemożliwiający dostęp do formularza, gdy istnieje oczekująca
+transakcja.
+
+Ostatnia grupa zadań nie została ukończona. Są to te same zadania, które nie zostały również ukończone w poprzedniej
+iteracji. Znajdują się w tej grupie zadania dotyczące pisania rozdziałów pracy dyplomowej, implementacji testów E2E,
+a także zadania związane z funkcjonalnościami portfela i płatności.
+
 ==== Podsumowanie
+
+Jest to kolejna już iteracja, w której nie ukończono bardzo wielu z zaplanowanych zadań. Pomimo przeciwności udało się
+wznowić prace nad aplikacją mobilną. Problemu te są częściowym usprawiedliwieniem w niepowodzeniach tej iteracji, gdyż
+pochłonęły one prawie połowę iteracji. Poczyniono nieznaczne postępy przy zadaniach związanych z serwerem i aplikacją
+mobilną. Nie ukończono natomiast żadnego zadania dotyczącego testów E2E, a także wielu zadań dotyczących aplikacji
+mobilnej oraz internetowej.
+
+.Wykres spalania sprintu dziewiątego.
+image::../images/sprints_raports/burndown_sprint9.png[]
+
+Udało się zakończyć 8 z 19 zadań wartych 19 punktów. Po raz kolejny wiele zadań zostało przeniesionych do następnej
+iteracji oraz do rejestru zadań. Nie jest to satysfakcjonujący wynik, natomiast jest i tak o wiele lepszy od
```

Comment:
czas

Reply:
ten wynik cały czas jest kiepski.

Reply:
w teorii tak, ale według mnie opisujemy to co się wydarzyło, wiec powinniśmy się trzymać tam czasu przeszłego. Nie wiem, może się mylę, trzeba by dopytać Pana Tadeusza

Reply:
najwyżej się to poprawi po jego review